### DISCUSSION TITLE
Can't get bdfr to authenticate new user

### DISCUSSION THREAD

**Discussion by FowadIjaz** [Serene-Arc/bulk-downloader-for-reddit#599] [Q&A] [answered]
Hi all, great tool, I'm trying to use a different user than the one I've been using previously, but when I do it returns 403 forbidden when it should redirect me to a link to authenticate according to the docs. I've tried with and without the --authenticate tag and the script crashes both ways. 

Here's the command I'm using:

python -m bdfr clone ./Test --user newuser --upvoted --authenticate --limit 40 --file-scheme '{POSTID},{REDDITOR},{TITLE}' --no-dupes --search-existing

I've tried going to the default_config.cfg file in Appdata\Local\BDFR\bdfr and moving it out, because I figured if it's not already there the script would autogenerate it, but that didn't appear to work. I've tried using venvs to run the script, but they seem to pull from the default_config.cfg file as well. I tried adding a config.cfg file to the folder with nothing in it, and it threw an error saying [DEFAULT] must be specified. I tried copying in all the parameters and setting them to NaN but that also didn't work. 

Appreciate any help...

**Comment by Serene-Arc** [comment, marked as answer]
If you open a bug report I can fix the real cause of this issue, which is that it doesn't matter what you supply vis-a-vis user authentication if there's a token in the config, which is definitely a bug.

In the meantime, you can use the `--config` option to specify a new configuration file. Copy the config file here on GitHub if needed. It should force the BDFR to reauthenticate and use the new user token.

**Reply by FowadIjaz** [reply]
Thanks, I've created an issue. In the meantime, I want to use the --config option and copy a config file from the repo here but can't find it. Can you please point me to it?

Edit: nevermind, I figured out that supplying `--log path/to/default_config.cfg` caused the authentication link to be run properly
